CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if (NOT MINGW)
   set(NOT_MINGW ON)
 else()
   set(NOT_MINGW OFF)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -static -static-libgcc -static-libstdc++")
```

Comment:
Doesn't this belong in the CMake toolchain file.

Reply:
I was thinking it also made sens for anyone building with MSYS2, is it not affected by this line?